activation/e2e/nipost_test.go

```diff
@@ -176,173 +187,44 @@ func TestNIPostBuilderWithClients(t *testing.T) {
 
 	t.Cleanup(launchPostSupervisor(t, logger, mgr, sig, grpcCfg, opts))
 
+	var postClient activation.PostClient
 	require.Eventually(t, func() bool {
-		_, err := svc.Client(sig.NodeID())
+		var err error
+		postClient, err = svc.Client(sig.NodeID())
 		return err == nil
 	}, 10*time.Second, 100*time.Millisecond, "timed out waiting for connection")
 
-	localDB := localsql.InMemory()
-	nb, err := activation.NewNIPostBuilder(
-		localDB,
-		poetDb,
-		svc,
-		[]types.PoetServer{{Address: poetProver.RestURL().String()}},
-		logger.Named("nipostBuilder"),
-		poetCfg,
-		mclock,
-	)
-	require.NoError(t, err)
-
-	challenge := types.NIPostChallenge{
-		PublishEpoch: postGenesisEpoch + 2,
-	}
-	nipost, err := nb.BuildNIPost(context.Background(), sig, &challenge)
+	post, info, err := postClient.Proof(context.Background(), shared.ZeroChallenge)
 	require.NoError(t, err)
 
-	v := activation.NewValidator(nil, poetDb, cfg, opts.Scrypt, verifier)
-	_, err = v.NIPost(
-		context.Background(),
-		sig.NodeID(),
-		goldenATX,
-		nipost.NIPost,
-		challenge.Hash(),
-		nipost.NumUnits,
-	)
-	require.NoError(t, err)
-}
-
-func TestNIPostBuilder_Close(t *testing.T) {
```

Comment:
Why was this test deleted? It verifies that creating a post can be canceled, something that wasn't possible at some time ago (or only with a big delay).

Reply:
It was not deleted but moved outside of end-to-end tests, where it doesn't belong.